src/analytics/lambdas/load-data-workflow/check-load-status.ts

```diff
@@ -67,53 +67,50 @@ export const handler = async (event: CheckLoadStatusEvent, context: Context) =>
   logger.debug(`query_id:${queryId}`);
   // There is a loading job need to check result.
   const response = await checkLoadStatus(queryId);
-
-  if (response.Status == StatusString.FINISHED) {
-    logger.info('Load success and delete the job in Dynamodb.');
-    for (let index = 0; index < jobList.entries.length; index++) {
-      const url = jobList.entries[index].url;
-      logger.debug(`delFinishedJobInDynamodb s3Uri:${url}`);
+  let errMsg: string | undefined = '';
```

Comment:
should we change it to `let errMsg = '';`, `errMsg` always is a string

Reply:
done